compiler/rustc_middle/src/ty/context.rs

```diff
@@ -2705,6 +2708,17 @@ impl<'tcx> TyCtxt<'tcx> {
         }
     }
 
+    pub fn mk_const_list<I: InternAs<[ty::Const<'tcx>], &'tcx List<ty::Const<'tcx>>>>(
```

Comment:
```suggestion
    pub fn mk_const_list<I: InternAs<ty::Const<'tcx>, &'tcx List<ty::Const<'tcx>>>>(
```
and will need to rebase so that this is ontop of #104533